spyder/plugins/variableexplorer/plugin.py

```diff
@@ -47,7 +47,19 @@ def get_icon(self):
         return self.create_icon('dictedit')
 
     def on_initialize(self):
-        pass
+        widget = self.get_widget()
+        widget.sig_open_preferences_requested.connect(
+            self._open_interpreter_preferences
```

Comment:
```suggestion
            self._open_preferences
```

The name is not correct.